src/sema/builtin_structs.rs

```diff
@@ -0,0 +1,199 @@
+                    indexed: false,
+                    readonly: true,
+                    recursive: false,
+                },
+            ],
+            offsets: Vec::new(),
+            storage_offsets: Vec::new(),
+        },
+        StructDecl {
+            tags: Vec::new(),
+            loc: pt::Loc::Builtin,
+            contract: None,
+            name: "AccountMeta".to_string(),
+            fields: vec![
+                Parameter {
+                    loc: pt::Loc::Builtin,
+                    id: Some(pt::Identifier {
+                        name: String::from("pubkey"),
+                        loc: pt::Loc::Builtin,
+                    }),
+                    ty: Type::Ref(Box::new(Type::Address(false))),
+                    ty_loc: None,
+                    indexed: false,
+                    readonly: false,
+                    recursive: false,
+                },
+                Parameter {
+                    loc: pt::Loc::Builtin,
+                    id: Some(pt::Identifier {
+                        name: String::from("is_writable"),
+                        loc: pt::Loc::Builtin,
+                    }),
+                    ty: Type::Bool,
+                    ty_loc: None,
+                    indexed: false,
+                    readonly: false,
+                    recursive: false,
+                },
+                Parameter {
+                    loc: pt::Loc::Builtin,
+                    id: Some(pt::Identifier {
+                        name: String::from("is_signer"),
+                        loc: pt::Loc::Builtin,
+                    }),
+                    ty: Type::Bool,
+                    ty_loc: None,
+                    indexed: false,
+                    readonly: false,
+                    recursive: false,
+                },
+            ],
+            offsets: Vec::new(),
+            storage_offsets: Vec::new(),
+        },
+        StructDecl {
+            tags: Vec::new(),
+            name: "ExternalFunction".to_string(),
+            loc: pt::Loc::Builtin,
+            contract: None,
+            fields: vec![
+                Parameter {
+                    loc: pt::Loc::Builtin,
+                    id: None,
+                    ty: Type::Bytes(4),
+                    ty_loc: None,
+                    indexed: false,
+                    readonly: false,
+                    recursive: false,
+                },
+                Parameter {
+                    loc: pt::Loc::Builtin,
+                    id: None,
+                    ty: Type::Address(false),
+                    ty_loc: None,
+                    indexed: false,
+                    readonly: false,
+                    recursive: false,
+                },
+            ],
+            offsets: Vec::new(),
+            storage_offsets: Vec::new(),
+        },
+    ]
+});
+
+impl StructType {
+    pub fn get_definition<'a>(&'a self, ns: &'a Namespace) -> &StructDecl {
```

Comment:
How about calling this `fn definition()`? I think `get_` is usually superfluous (and the word terribly overloaded in the English language anyway).